### PR TITLE
COMMERCE-12357 Edit UOM in the mini-cart + Automation

### PR DESCRIPTION
**Pull request by baharturk**
Story:
https://liferay.atlassian.net/browse/COMMERCE-11668

Test Plan:
https://liferay.atlassian.net/browse/COMMERCE-12597

sent from: #4297 

@smottal

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+				uomKey = "uom3",
+				uomName = "UOM3");
+
+			for (var uomKey : list "uom2,uom3") {
+				CommerceJSONWarehousesAndInventoriesAPI._patchWarehouseItem(
+					productSku = "MIN93015",
+					quantity = 10,
+					uomKey = ${uomKey},
+					warehouseName = "Italy");
+			}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase:273]: if back order is enabled, there's no need to set stock quantity

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+				cartItemQuantity = "0.6",
+				cartItemSku = "MIN93015",
+				cartItemUOMKey = "uom2",
+				cartSummaryTotal = "2.00");
+		}
+	}
+
+	@description = "COMMERCE-12619. As a user, I can verify different pricing types are applied correctly upon editing UOM in the mini cart."
+	@priority = 5
+	test CanVerifyDifferentPricingTypesAppliedCorrectly {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase:229]: test CanAssertPricingTypesAreAppliedWhileEditingUOMInMiniCart {

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+
+			Click(locator1 = "CommerceAccelerators#QUICK_ADD_TO_CART_BUTTON");
+		}
+
+		task ("When clicks the Edit from the ellipsis in the mini cart") {
+			Click(
+				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS_WITH_UOM",
+				uomKey = "uom1");
+
+			DropdownMenuItem.click(menuItem = "Edit");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase:319]: Can you create a navigation macro for this? Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+					key_priceField = ${priceField},
+					key_priceValue = ${priceValue},
+					locator1 = "CommerceAccelerators#MINI_CART_EDIT_ITEM_UOM_PRICE_VALUE");
+			}
+		}
+	}
+
+	@description = "COMMERCE-12620. As a user, I can verify price table information in mini cart's UOM edit section."
+	@priority = 5
+	test CanVerifyPriceTableInMiniCartEditUOMSection {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCartEditUnitsOfMeasure.testcase:404]: test CanVerifyPriceTableWhileEditingUOMInMiniCart {

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:test:stable

**Comment by baharturk** [comment]
ci:test:stable

**Comment by baharturk** [comment]
ci:test:stable

**Comment by baharturk** [comment]
The failures on the relevant are not related to the PR, this can be forwarded. Thanks!

@andrea-ale-sbarra @luca-pellizzon

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Pull request closed without merging**